### ISSUE TITLE
High accuracies on training and validation but poor performance on test set evaluation with autokeras version 0.2.11

### ISSUE THREAD

**Issue by swenkel** [jhfjhfj1/autokeras#168]
### Bug Description

Auto-Keras version 0.2.11 (installed from pip) leads to models with cross-validation accuracies > 0.97 after a few minutes of training.  The results of the test set is always around 10 %. I exported and loaded the model to keras manually and reached ~ 10 % as well.

**This problem is solved with the current develop branch and if I look at [this video example](https://www.youtube.com/watch?v=Hsw9z1GcAms) it was not there before.**

### Reproducing Steps
  * Step 1: Running the mnist example from the documentation will yield something around 10 % acc independent of the training time limit (0.5 - 7 h) using autokeras 0.2.11 installed from pip.
  * Step 2: Running the mnist example from the documentation will yield something around >9x % acc independent of the training time limit using autokeras develop.

### Expected Behavior

Returning accuracies not too far from cross-validation acc.

### Setup Details
Include the details about the versions of:
and version: Linux
 - Python: 3.6.5
 - autokeras: 0.2.11
 - scikit-learn: 0.19.1
 - numpy: 1.14.5
 - keras: 2.2.2
 - scipy: 1.1.0
 - tensorflow: 1.10.0
 - pytorch: 0.4.1
### Additional context

Release the current develop on pip and this problem is solved.

**Comment by jhfjhfj1** [comment]
@swenkel 
Thank you for your report!
The bug is fixed in the new release.
